remote-config.jsonc

```diff
@@ -60,12 +60,18 @@
 				{
 				    "message": "Platform.sh sponsors DDEV by paying its lead maintainer to work on DDEV full time!"
 				},
+				{
+				    "message": "Need to get the database and files from your Platform.sh stack? Try `ddev pull platform`! https://ddev.readthedocs.io/en/latest/users/providers/platform"
+				},
 				{
 					"message": "Use `ddev composer` instead of `composer` to ensure your project uses the Composer and PHP versions it specifies."
 				},
 				{
 					"message": "`ddev npm` is the right way to run npm commands in your web container."
 				},
+				{
+					"message": "`ddev yarn` is the right way to run yarn commands in your web container."
```

Comment:
```suggestion
					"message": "`ddev yarn` is a great way to run yarn commands in your web container."
```

Reply:
I copied this from the NPM command above ("the right way"), for consistency, I'd say either have both "a great way" or both "the right way"?

Reply:
"the right way" just seemed kind of prescriptive. Not sure it matters. But using `yarn` inside the container or using `ddev exec yarn`... nothing wrong with any of those.